docs/tutorials/adding-a-support-desk-button.md

```diff
@@ -101,10 +122,13 @@ export default () => {
 };
 ```
   <template v-slot:caption>
-    modules/asset/ui/src/index.js
+    modules/support-desk/ui/apos/apps/SupportDesk.js
   </template>
 </AposCodeBlock>
 
+> [!IMPORTANT]
+> Note the location of this script. It should be placed in the `ui/apos/apps` folder of your module in order to have access to the `apos.bus.$on()`.
```

Comment:
and to avoid loading it for logged-out site visitors who do not need the extra overhead

Reply:
Added.